loopchain/blockchain/blockchain.py

```diff
@@ -98,6 +111,60 @@ def __init__(self, channel_name=None, store_id=None, block_manager=None):
         for tx_version, tx_hash_version in channel_option.get("hash_versions", {}).items():
             self.__tx_versioner.hash_generator_versions[tx_version] = tx_hash_version
 
+    @property
+    def leader_made_block_count(self) -> int:
+        if self.__last_block:
+            return self.__made_block_count[self.__last_block.header.peer_id]
+        return -1
+
+    @property
+    def my_made_block_count(self) -> int:
+        return self.__made_block_count[ChannelProperty().peer_address]
+
+    def _up_made_block_count(self, block: Block) -> None:
```

Comment:
just suggestion, if you intended increasing the value of `__made_block_count`.
it is also go along with `reset_leader_made_block_count` and other methods, which are consisted of "verb" and its "object".

```suggestion
    def _increase_made_block_count(self, block: Block) -> None:
```